activity-metrics-list.md

```diff
@@ -44,6 +45,7 @@ We list and describe activity metrics. For different situations, the metrics hav
 |Non-Source Contributions|Track contributions like running tests in test environment, writing blog posts, producing videos, giving talks, etc...|
 |Onion Layers|*Distance between onion model layers (users, contributors, committers, and steering committee)*\\ Rule of thumb: factor of 10x between layers. (OSLS'17 Node.js keynote)|
 |Path to Leadership|*A communicated path from lurker to contributor to maintainer. (or. track members: time from user to maintainer/leader)*\\ Rational: If active contributors are not included in leadership decisions they might lose interest and leave. (Focus on least likely contributor)|
+|Pony Factor|The minimum number of developers performing 50% of the commits. [The Math](https://ke4qqq.wordpress.com/2015/02/08/pony-factor-math/)|

```

Comment:
There's a distinction between the Pony Factor (PF) and the Augmented Pony Factor (APF) on that thread. The APF only takes in account currently active developers. I'm not sure if you want to distinguish those, and whether we have a clear definition of what "currently active" means.

Reply:
@tobie, do you think that we have to determine 'currently active'? Perhaps, it could be a judgement by folks using the tools. We could set an arbitrary threshold but I agree, it would be arbitrary.

Reply:
I guess the definition could be "it's defined on per project basis." Still would be useful to define the term, imho. But then I also edit specs as a job, so I might have a higher level of formalism baked in me than you want to aim for with this project.